scripts/python/testbedApi.py

```diff
@@ -0,0 +1,329 @@
+# Python testing utility classes and functions
```

Comment:
hah. well, I still think if we can avoid introducing new tools, that we should avoid it. The biggest problem with python in our code is that python on the host will differ wildly, anywhere from 2.x to 3.x (which as I understand it are wildly different languages).

To keep a quality build experience we check the go env before we build. We could do this for python too, and in fact I would expect it if we keep this code.

Perhaps this would be better put in a contrib, etc dir in the systemtests-utils library, which is where the systemtests are going,  see #143 .